OrbitGl/ImGuiOrbit.h

```diff
@@ -20,13 +20,13 @@
 #include "imgui.h"
 #include "imgui_internal.h"
 
-#define IMGUI_VAR_TO_TEXT(var) ImGuiOrbit::VariableToText(#var, var)
-#define IMGUI_VARN_TO_TEXT(var, name) ImGuiOrbit::VariableToText(name, var)
+#define IMGUI_VAR_TO_TEXT(var) orbit_gl::imgui::VariableToText(#var, var)
+#define IMGUI_VARN_TO_TEXT(var, name) orbit_gl::imgui::VariableToText(name, var)
 
 #define IMGUI_FLOAT_SLIDER(x) IMGUI_FLOAT_SLIDER_MIN_MAX(x, 0, 100.f)
 #define IMGUI_FLOAT_SLIDER_MIN_MAX(x, min, max) ImGui::SliderFloat(#x, &x, min, max)
 
-namespace ImGuiOrbit {
+namespace orbit_gl::imgui {
```

Comment:
Seeing now that this namespace seems to contain only this one function, I would just keep `orbit_gl` only and if we really want to just rename the function to `VariableToImguiText`?

Reply:
Done.